cli/internal/config/validateAudit_test.go

```diff
@@ -0,0 +1,394 @@
+package config
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestValidateAudit(t *testing.T) {
+	tests := []struct {
+		name        string
+		cfg         *Config
+		expectError bool
+		errorMsg    string
+	}{
```

Comment:
Something to consider: when you have a deeply nested struct in a test array it can be nice to either construct it using an inline util function or split the arrays into 2 arrays so that the list of cases is easier to see.